carbon-design-system/carbon-addons-iot-react
# fix(card): accessibility violation in card header tooltip

#3395

Closes #3395

**Summary**

- Card with a tooltip in the header throws serious accessibility violation for missing aria-label.
<img width="795" alt="image" src="https://user-images.githubusercontent.com/22093326/159900713-4a34e061-1b0d-4c22-9029-b9065679eb63.png">


**Change List (commits, features, bugs, etc)**

- Setting the `iconDescription` prop of `CardTitle`, sets the aria-label to the card header tooltip div and hence gets rid of accessibility violation.

**Acceptance Test (how to verify the PR)**

- Go to storybook /?path=/story/1-watson-iot-card-card--with-ellipsed-title-tooltip-external-tooltip
- Check the Accessibility tab
- The serious violation 'Ensures every ARIA button, link and menuitem has an accessible name' is no more listed.

**Regression Test (how to make sure this PR doesn't break old functionality)**

- As there is no functionality changes, so the Card storybook and all components referring Card should work as before.



**Things to look for during review**
